### DISCUSSION TITLE
Время в системе

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#229] [ОШБК!] [answered]
Точно помню, что при автоматической установке системы на сервер указал свою страну и часовой пояс. Но по итогу когда подставляется текущее время, то оно неверно. Подскажите, где искать причину проблемы? 

https://prnt.sc/uJvLlOUuUO3D

**Comment by ttm-alexeypolunin** [comment]
Неизвестно что там хетцнер накрутил с образом — напишите мне в телеграм.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Была ошибка в установщике без SSL. Если вдруг вы столкнулись с такой же ошибкой, то вам нужно:

1. Зайти на сервер под root.

2. Вывести свою таймзону
```
timedatectl
```

3. Открыть на редактирование файл конфигурации пула fpm:
```
nano /etc/php/8.0/fpm/pool.d/totum.conf
```

4. Заменить в нем `Europe/London` на правильную таймзону, скопированную из предыдущего шага и сохранить изменения `Cntrl` + `X` ввести `Y` далее `Enter`.

5. Перезапустить fpm:
```
service php8.0-fpm restart
```

**Reply by mihail-cornev** [reply]
Скажите пожалуйста, а потом нужно будет сносить сервер и ставить заново, чтобы работала отправка писем и для подключения домена?

**Reply by ttm-alexeypolunin** [reply]
Можно, но мы не консультируем по этому вопросу тк это за пределами наших бесплатных возможностей. Можем вас переключить за $100

**Reply by mihail-cornev** [reply]
> Можно, но мы не консультируем по этому вопросу тк это за пределами наших бесплатных возможностей. Можем вас переключить за $100

спасибо
мне главное знать, что это можно сделать без сноса ))  чтобы мы могли какие-то данные вводить и тестировать, потихоньку начинать работать в системе.
